apps/web/components/ui/form/Select.tsx

```diff
@@ -30,7 +30,7 @@ function Select<
   Group extends GroupBase<Option> = GroupBase<Option>
 >({ className, ...props }: SelectProps<Option, IsMulti, Group>) {
   const [mounted, setMounted] = useState<boolean>(false);
-  const { resolvedTheme, forcedTheme } = useTheme();
+  const { resolvedTheme, forcedTheme } = useGetTheme();
```

Comment:
`useTheme` by default won't return these values. This is to avoid accidentally modifying theme of a page where the purpose is just to get the values.
So, we have a separate hook to just get the values.